entrypoint.sh

```diff
@@ -15,6 +15,7 @@ export DD_CONF_SUPERVISOR_SOCKET="/dev/shm/datadog-supervisor.sock"
 
 
 ##### Core config #####
+sed -i '/^#.* \[trace.sampler\]/s/^# //' ${DD_ETC_ROOT}/datadog.conf
```

Comment:
Instead of adding the section here, what do you think about patching `set_property` in config_builder.py to do
```python
if not self.config.has_section(section):
    self.config.add_section(section)
```
See https://docs.python.org/2/library/configparser.html#ConfigParser.RawConfigParser.add_section

I do think failing in `config_builder.py` because the section does not exist is a bug.

Reply:
Good idea - much better than my original hack.  Done!